dist/stat_data.py

```diff
@@ -242,11 +242,19 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     BlockStat('block_byte_map_read', 'mapped bytes read', 'size'),
     BlockStat('block_byte_read', 'bytes read', 'size'),
+    BlockStat('block_byte_read_intl', 'bytes read for internal pages', 'size'),
+    BlockStat('block_byte_read_intl_precomp', 'bytes read for internal pages before decompression and decryption', 'size'),
```

Comment:
precomp is a bit weird. May be rename to "block_byte_read_intl_disk"?

Reply:
Yeah that is fair, I've renamed it